pkg/katautils/create.go

```diff
@@ -204,6 +204,16 @@ func CreateSandbox(ctx context.Context, vci vc.VC, ociSpec oci.CompatOCISpec, ru
 		return nil, vc.Process{}, err
 	}
 
+	defer func() {
+		// cleanup netns if kata creates it
+		ns := sandboxConfig.NetworkConfig
+		if err != nil && ns.NetNsCreated {
+			if ex := cleanupNetNS(ns.NetNSPath); ex != nil {
+				kataUtilsLogger.WithField("path", ns.NetNSPath).WithError(ex).Warn("Failed to cleanup netns")
```

Comment:
You might find that the CI linters complain as errors should start with a lower-case letter (same comment for errors below...)

Reply:
I was vary confuse about this, if my memory is right, golang advised log should be start begin with lower-case. But kata use them both, that confused me, so I followed the upper-case way.
Make all log begin with lower-case?

Reply:
Hi @Ace-Tang - sorry! I meant to put this comment on the first of your new `fmt.Errorf()` calls as this clearly isn't an error, but a log call that's logging an error ;)

But yes, I think you are correct - we need to be more consistent and make all errors start with a lower-case letter at some point (not this PR).

But, for now, if the CI is happy, I'm happy :smile:

Reply:
Hi, @jodh-intel ,do you mean 
```
n, err := ns.GetNS(netNSPath)
187     if err != nil { 
188         return fmt.Errorf("Failed to get netns %s: %v", netNSPath, err)   --> change Fail to fail, is it ?
189     }
```

Reply:
Yes. As I say, if the CI checks are happy, no need to change strictly. But I think this should probably be:

```go
return fmt.Errorf("failed to get netns %s: %v", netNSPath, err)
```

Reply:
Update, @jodh-intel